ramls/loan.json

```diff
@@ -53,6 +53,18 @@
       "description": "ID of last policy used in relation to this loan",
       "type": "string"
     },
+    "loanPolicy": {
+      "description": "Additional information about the loan policy of the item, taken from the loan loanPolicyId",
+      "readonly": true,
+      "type": "object",
+      "properties": {
+        "loanPolicyName": {
```

Comment:
Given this is within a `loan policy` object, it can be called `name` rather than `loan policy name`

Reply:
While this is reasonable, there are other parts of the system, where the loanPolicyName property is used, for example, in ValidationError

Reply:
Indeed, that error message is a rather different context to record design. That parameter is only really presented as an aid to client, and we do not have the ability to nest parameters (we could use something like JSON path, however that is a more significant discussion).